jekyll-assets/_includes/subtitle404.html

```diff
@@ -0,0 +1,3 @@
+<div class="subtitle">
+  <p>We couldn't find what you were<br /> looking for. Please browse our updated<br /> <strong>Raspberry Pi</strong> Documentation.</p>
```

Comment:
@nelliemckesson I put in these explicit `<br />` so that it matched Jack's mockup image. But I guess it's probably "better" to simply make this a fixed-width paragraph, and let the browser do the text-reflowing? (but again that's more CSS knowledge than I'm capable of)